onebusaway-android/src/main/java/org/onebusaway/android/report/ui/InfrastructureIssueActivity.java

```diff
@@ -172,29 +174,41 @@
     /**
      * Starts the InfrastructureIssueActivity.
      *
-     * @param activity The context of the activity.
+     * @param activity The parent activity.
      * @param intent   The Intent containing focusId, lat, lon of the map
      */
     public static void start(Activity activity, Intent intent) {
-        activity.startActivityForResult(makeIntent(activity, intent), 0);
+        activity.startActivityForResult(makeIntent(activity, intent), REQUEST_CODE);
     }
 
     /**
```

Comment:
@cagryInside Could you add a little more information to the Javadoc here of how `.startActivityForResult()` is used (e.g., to close the calling activity)?  For future developers it would be good to document the difference in behavior here from the one that takes a `Context` as a parameter.